retrofit-mock/src/main/java/retrofit2/mock/Calls.java

```diff
@@ -137,6 +137,7 @@ public Call<T> clone() {
     }
 
     @Override
+    @SuppressWarnings("return") //Suppressed for purpose of time and help. The warning must still be examined. 
```

Comment:
What's the error here? I thought that `request()` was `@MustCallAlias` with the response, so I would expect the result of this method to be must-call aliased with response, too. But maybe that's not possible because of the possibility that `response` is null? 

Can you comment with the error here? That will help me reason through this.